starter-code/server.js

```diff
@@ -106,7 +111,7 @@ app.put('/articles/:id', function(request, response) {
 
 app.delete('/articles/:id', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Corresponds with number 3 of the diagram. truncateTable on line 58. part of Delete in CRUD.
```

Comment:
the method in article.js is `deleteRecord` because this is deleting a single article not the entire table